### ISSUE TITLE
datalad.core.distributed.tests.test_clone.test_clone_report_permission_issue fails with adjusted branches

### ISSUE THREAD

**Issue by mih** [datalad/datalad#4226]
On Debian, as of current master (7b8282b5d671c45a0c4ae3c42024edf27ce51230), with TMPDIR on a crippled FS.
git-annex version: 8.20200226; git version 2.24.1

```
======================================================================
FAIL: datalad.core.distributed.tests.test_clone.test_clone_report_permission_issue
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/mih/env/datalad3-dev/lib/python3.7/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/mih/hacking/datalad/git/datalad/tests/utils.py", line 222, in newfunc
    return func(*args, **kwargs)
  File "/home/mih/hacking/datalad/git/datalad/tests/utils.py", line 759, in newfunc
    return t(*(arg + (filename,)), **kw)
  File "/home/mih/hacking/datalad/git/datalad/core/distributed/tests/test_clone.py", line 378, in test_clone_report_permission_issue
    assert_status('error', res)
  File "/home/mih/hacking/datalad/git/datalad/tests/utils.py", line 1313, in assert_status
    dumps(results, indent=1, default=lambda x: str(x))))
AssertionError: Test 1/1: expected status ['error'] not found in:
[
 {
  "action": "install",
  "path": "/media/mih/Samsung_T5/tmp/datalad_temp_test_clone_report_permission_issueudwnxgzm/protected/datasets-tests.datalad.org",
  "type": "dataset",
  "status": "ok",
  "source_url": "///",
  "source": {
   "source": "///",
   "version": null,
   "type": "dataladri",
   "giturl": "http://datasets-tests.datalad.org/.git",
   "default_destpath": "datasets-tests.datalad.org"
  }
 }
]
```

**Comment by mih** [comment]
Still happening as of now.

**Comment by mih** [comment]
Fixed in c287e33596a1ace202e7b54c81c36e6a9aa681a6
